cmd/bridge/main.go

```diff
@@ -401,13 +407,40 @@ func createPgDb(cfg *bridgeCore.Config) {
 	}
 }
 
+func setupStats(cfg *bridgeCore.Config, db *gorm.DB) {
+	// check if bridge stats is enabled
+	if os.Getenv(bridgeStatsNodeName) != "" && os.Getenv(bridgeStatsUrl) != "" {
+		// NOTE: only get chainId, operator from ronin
+		var (
+			chainId, operator string
+			node              = os.Getenv(bridgeStatsNodeName)
+			host              = os.Getenv(bridgeStatsUrl)
+			pass              = os.Getenv(bridgeStatsSecret)
+		)
+		if _, ok := cfg.Listeners[RoninNetwork]; ok {
+			chainId = cfg.Listeners[RoninNetwork].ChainId
+			if cfg.Listeners[RoninNetwork].Secret != nil && cfg.Listeners[RoninNetwork].Secret.BridgeOperator != nil {
+				signMethod, err := bridgeCoreUtils.NewSignMethod(cfg.Listeners[RoninNetwork].Secret.BridgeOperator)
+				if err != nil {
+					panic(err)
+				}
+				operator = signMethod.GetAddress().Hex()
+			}
+		}
+		stats.NewService(node, chainId, operator, host, pass, db)
+		go stats.BridgeStats.Start()
+	}
+}
+
 func bridge(ctx *cli.Context) {
 	cfg := prepare(ctx)
 	// init db
 	db, err := bridgeCoreStore.MustConnectDatabase(cfg.DB, false)
 	if err != nil {
 		panic(err)
 	}
+	// setup stats
+	setupStats(cfg, db)
```

Comment:
I think setupStats should be placed after db migration to prevent querying error